app/models/user.js

```diff
@@ -65,5 +65,7 @@ module.exports = (sequelize, DataTypes) => {
   User.makeAdmin = email =>
     User.update({ isAdmin: true }, { where: { email } }).catch(error => databaseError(error.message));
 
+  User.getUserById = id => User.findOne({ where: { id } }).catch(error => databaseError(error.message));
```

Comment:
Just so you know, sequelize provides you with a `getById(id: int)` for every model!
You can change it in the next PR.